doc/Metrics.md

```diff
@@ -62,12 +64,65 @@ added to the deployment with the following parameters:
   "version": "1.0"
 }
 ```
-After the modules are deployed, Chronograf can be reached at localhost:8888.
+After the modules are deployed, Chronograf can be reached at http://localhost:8888. InfluxDb should be configured as a data source for Chronograf. The InfluxDb instance should be reachable at http://influxdb:8086.
 
 ## Example InfluxDb queries
 
 For example InfluxDb queries, please look at https://github.com/Azure/iotedge/pull/141/files#diff-9fc75ceaff4ae01d42e1bafad8037848
 
 ## Collecting CPU and memory stats from EdgeHub and other containers
 
-A telegraf container can be added to the deployment to collect stats about all containers in the deployment. Please refer to https://hub.docker.com/_/telegraf/ for steps to configure it.  
+A telegraf container can be added to the deployment to collect stats about all containers in the deployment. Telegraf uses a configuration file on the device to configure itself. Azure IoT Edge deployments can only mount files in to a container but cannot pass a configuration file to the device from the cloud. For this reason, the configuration has to be done on the device as follows:
+```
+mkdir telegraf
+docker run --rm telegraf telegraf config > telegraf/telegraf.conf
+```
+Edit the file to reflect the following changes:
+```
+[[outputs.influxdb]]
+    urls = ["http://influxdb:8086"]
+[[inputs.docker]]
+  endpoint = "unix:///var/run/docker.sock"
+```
+The telegraf container can be run manually on the device as follows:
+```
+docker run -d --name=telegraf --net=azure-iot-edge -v /var/run/docker.sock:/var/run/docker.sock -v $PWD/telegraf/telegraf.conf:/etc/telegraf/telegraf.conf:ro telegraf
+```
+Or the container can be added to an Azure IoT Edge deployment using the following deployment information:
+```
+"telegraf": {
+    "type": "docker",
+    "settings": {
+    "image": "telegraf",
+    "createOptions": "{\"HostConfig\":{\"Binds\":[\"/var/run/docker.sock:/var/run/docker.sock\",\"/home/jadsa/telegraf/telegraf.conf:/etc/telegraf/telegraf.conf\"]}}"
```

Comment:
Does it require the Docker socket to be bind mounted as well?

Reply:
Yes. It collects stats from the docker socket.